uboot/uboot.defconfig

```diff
@@ -1,74 +1,68 @@
 CONFIG_ARM=y
+CONFIG_SKIP_LOWLEVEL_INIT=y
 CONFIG_ARCH_AT91=y
 CONFIG_SYS_TEXT_BASE=0x26f00000
+CONFIG_SYS_MALLOC_F_LEN=0x8000
 CONFIG_TARGET_SAMA5D27_WLSOM1_EK=y
-CONFIG_SPL_GPIO_SUPPORT=y
-CONFIG_SPL_LIBCOMMON_SUPPORT=y
-CONFIG_SPL_LIBGENERIC_SUPPORT=y
-CONFIG_SYS_MALLOC_F_LEN=0x2000
-CONFIG_ENV_OFFSET=0x00100000
+CONFIG_ENV_OFFSET=0x100000
```

Comment:
This is weird. `0x00100000` and `0x100000` should be numerically equal. The old value doesn't boot though and the new one does.